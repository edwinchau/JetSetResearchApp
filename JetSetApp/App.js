import {
  createStackNavigator,
  createAppContainer
} from 'react-navigation';
import SurveyCompletedScreenExample from './app/screens/SurveyCompletedScreenExample/SurveyCompletedScreenExample';
import SurveyScreenExample from './app/screens/SurveyScreenExample/SurveyScreenExample';
import HomeScreenExample from "./app/screens/HomeScreenExample/HomeScreenExample";

// const stackNav = createStackNavigator({
//   Home: {
//     screen: HomeScreenExample
//   },
//   Survey: {
//     screen: SurveyScreenExample
//   },
//   SurveyCompleted: {
//     screen: SurveyCompletedScreenExample
//   },
// });

const stackNav = createStackNavigator(
    {
      Home: HomeScreenExample,
      Survey: SurveyScreenExample,
      SurveyCompleted: SurveyCompletedScreenExample,
    },
    {
      initialRouteName: 'Home',
    }
);


const AppContainer = createAppContainer(stackNav);

export default AppContainer;
