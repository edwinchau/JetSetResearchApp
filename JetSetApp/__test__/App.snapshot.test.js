import React from "react";
import App from "../App"
import renderer from "react-test-renderer"

describe("<App/>", ()=>{
  it("snapshot", async ()=>{
    expect(renderer.create(<App/>).toJSON()).toMatchSnapshot();
  });
});
