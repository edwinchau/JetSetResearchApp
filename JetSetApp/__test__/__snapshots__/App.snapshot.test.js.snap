// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rgba(108,48,237,1)",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View>
    <ActivityIndicator
      animating={true}
      color="#999999"
      hidesWhenStopped={true}
      size="small"
    />
  </View>
</View>
`;
